tasks/bower-concat.js

```diff
@@ -95,7 +96,7 @@ module.exports = function(grunt) {
 				var cssFiles = {};
 
 				_.each(lists.components, function(component, name) {
-					if (includes.length && _.indexOf(includes, name) === -1) return;
+					if (!options.includeWithDependencies && includes.length && _.indexOf(includes, name) === -1) return;
```

Comment:
But it will just ignore `include` if `includeWithDependencies` is true, won’t it?